tests/integration/data/interact_with_mlmd.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eux
+MODEL=$1
+echo "MODEL=$MODEL"
+wget https://raw.githubusercontent.com/google/ml-metadata/master/ml_metadata/proto/metadata_store.proto
+wget https://raw.githubusercontent.com/google/ml-metadata/master/ml_metadata/proto/metadata_store_service.proto
+wget -O- https://github.com/fullstorydev/grpcurl/releases/download/v1.8.0/grpcurl_1.8.0_linux_x86_64.tar.gz | tar -xzv
+mkdir -p ml_metadata/proto/
+mv metadata_store.proto ml_metadata/proto/
+SERVICE=$(kubectl get services/mlmd -n $MODEL -oyaml | yq e .spec.clusterIP -)
+./grpcurl -v --proto=metadata_store_service.proto --plaintext $SERVICE:8080 ml_metadata.MetadataStoreService/GetArtifacts
```

Comment:
Should I python-ify this?  The `kubectl` call could be the python k8s sdk, `wget`s could be in python, then `grpcurl` could be a `subprocess.run` (or maybe connect some other way)

Alternatively, we could use the [mlmd python api](https://www.tensorflow.org/tfx/guide/mlmd) and just call to it using the native ways in python

Reply:
(or should we table this for future and prioritize the 1.4 release first)

Reply:
Ideally, this seems like it should be a pytest fixture that does all the setup, and returns a `MetadataStore` (judging by my reading of the docs) object . I think it's also fine to punt on that for now and focus on 1.4.